example/onnx/super_resolution.py

```diff
@@ -1,86 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-"""Testing super_resolution model conversion"""
-from __future__ import absolute_import as _abs
-from __future__ import print_function
-from collections import namedtuple
-import logging
-import numpy as np
-from PIL import Image
-import mxnet as mx
-from mxnet.test_utils import download
-import mxnet.contrib.onnx as onnx_mxnet
-
-# set up logger
-logging.basicConfig()
-LOGGER = logging.getLogger()
-LOGGER.setLevel(logging.INFO)
-
-def import_onnx():
```

Comment:
this example is used in one of the ONNX tests - https://github.com/apache/incubator-mxnet/blob/master/tests/python-pytest/onnx/import/onnx_import_test.py#L152 If you are deleting this, you will have to remove the test too.

Reply:
The framework tests shouldn't depend on the examples. I will remove that test.

The nightly tests should run the tutorials test and raise an error if the onnx tutorial cannot run

Reply:
Agree, having a test for the correctness of this example was, I think, a review comment at the time. Thanks for removing it.

Reply:
updated, thanks for pointing it out @anirudhacharya

Reply:
That review comment was from my side. The idea was to introduce test that validate the examples. As of right now, almost all of our examples go entirely untested. But fine with me.